apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -288,10 +288,28 @@ data class Avkorting(
                                         tom = inntektsavkorting.grunnlag.periode.tom,
                                     )
 
+                                val inntekt =
+                                    when (inntektsavkorting.grunnlag.inntektInnvilgetPeriode) {
+                                        is BeregnetInntektInnvilgetPeriode -> inntektsavkorting.grunnlag
+                                        is IngenInntektInnvilgetPeriode ->
```

Comment:
Her burde vi kanskje hatt en liten test som fanger scenarioet og beskriver flyten for avkorting?